docs/examples/bar-chart.html

```diff
@@ -133,5 +143,8 @@ <h5 class="card-title">Grouped bar chart with dataset</h5>
         </div>
       </div>
     </div>
+    <!-- Cookies -->
+    <script crossorigin="anonymous" integrity="sha384-AcN5kJiCfqbrk/SICuAawqTeUR7KQrKpXMeDFL5XZh7x+IBWERL5to8nwE0gapz6" src="https://cdn.jsdelivr.net/npm/tarteaucitronjs@1.16.1/tarteaucitron.min.js"></script>
```

Comment:
@louismaximepiton Could you please update this PR to use tarteaucitron v1.17.0 (see https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2503)